lib/src/signed_video_tlv.h

```diff
@@ -150,10 +150,10 @@ uint8_t
 read_byte(uint16_t *last_two_bytes, const uint8_t **payload, bool do_emulation_prevention);
 
 /**
- * @brief Scans the TLV part of a SEI payload and decodes the found tag when it's detected.
+ * @brief Scans the TLV part of a SEI payload and decodes the found tag when it is detected.
  *
- * The data is assumed to have been written in a TLV format. tlv_find_tag parses data as long as
- * there are new tags, but never decodes it. The function can handle data both with and without
+ * The data is assumed to have been written in a TLV format. tlv_find_tag_and_decode parses data
+ * and finds correct tag and decodes it. The function can handle data both with and without
  * emulation prevention bytes.
```

Comment:
You should probably tell that SVI_INVALID_PARAMETER is returned if the tag is not found.